src/warehouse/app.py

```diff
@@ -102,3 +97,19 @@ def restrict_admin():
 
     # Add CORS information for all resources.
     CORS(application)
+
+    # Add an error handler for webargs parser error, ensuring a JSON response
+    # including all error messages produced from the parser.
+    @application.errorhandler(422)
```

Comment:
I don't quite follow why the error handlers are defined inside the `init_app` rather than in their own module.

Reply:
This is part of defining the flask app's behaviour. Since it was just a couple of functions, I inlined them, they might as well also go in a separate module for readability.